src/js/edu-benefits/reducers/uiState/index.js

```diff
@@ -0,0 +1,182 @@
+      fields: []
+    },
+    '/veteran-information/dependent-information': {
+      complete: false,
+      verified: false,
+      fields: []
+    },
+    '/veteran-information/direct-deposit': {
+      complete: false,
+      verified: false,
+      fields: []
+    },
+    '/review-and-submit': {
+      complete: false,
+      verified: false,
+      fields: []
+    }
+  },
+  panels: [
+    {
+      path: '/benefits-eligibility',
+      name: 'Benefits Eligibility',
+      classes: 'one',
+      sections: []
+    },
+    {
+      path: '/military-history',
+      name: 'Military History',
+      classes: 'two',
+      sections: [
+        { path: '/military-history/service', name: 'Military Service' },
+        { path: '/military-history/rotc-history', name: 'ROTC History' },
+        { path: '/military-history/benefits-history', name: 'Benefits History' }
+      ]
+    },
+    {
+      path: '/education-history',
+      name: 'Education History',
+      classes: 'three',
+      sections: []
+    },
+    {
+      path: '/employment-history',
+      name: 'Employment History',
+      classes: 'four',
+      sections: []
+    },
+    {
+      path: '/school-selection',
+      name: 'School Selection',
+      classes: 'five',
+      sections: []
+    },
+    {
+      path: '/veteran-information',
+      name: 'Veteran Information',
+      classes: 'six',
+      sections: [
+        { path: '/veteran-information/personal-information', name: 'Personal Information' },
+        { path: '/veteran-information/address', name: 'Address' },
+        { path: '/veteran-information/contact', name: 'Contact Information' },
+        { path: '/veteran-information/secondary-contact', name: 'Secondary Contact' },
+        { path: '/veteran-information/dependent-information', name: 'Dependent Information' },
+        { path: '/veteran-information/direct-deposit', name: 'Direct Deposit' },
+      ]
+    },
+    {
+      path: '/review-and-submit',
+      name: 'Review',
+      classes: 'seven last',
+      sections: []
+    }
+  ]
+};
+
+function uiState(state = ui, action) {
+  switch (action.type) {
+    case UPDATE_COMPLETED_STATUS:
+      return _.set(['sections', action.path, 'complete'], true, state);
+
+    case UPDATE_INCOMPLETE_STATUS:
+      return _.set(['sections', action.path, 'complete'], false, state);
+
+    case UPDATE_REVIEW_STATUS:
+      return _.set(['sections', action.path, 'complete'], action.value, state);
+
+    case UPDATE_VERIFIED_STATUS:
+      return _.set(['sections', action.path, 'verified'], action.value, state);
+
+    case UPDATE_SUBMISSION_STATUS:
+      return _.set(['submission', action.path, 'status'], action.value, state);
+
+    case UPDATE_SUBMISSION_ID:
+      return _.set(['submission', action.path, 'id'], action.value, state);
+
+    case UPDATE_SUBMISSION_TIMESTAMP:
+      return _.set(['submission', action.path, 'timestamp'], action.value, state);
+
+    default:
+      return state;
+  }
+}
+
+export default uiState;
```

Comment:
Yeah, that'd make sense. I could just remove them for now, since they're not used yet. But presumably we will.